msteams-platform/build-your-first-app/build-bot.md

```diff
@@ -1,95 +1,117 @@
 ---
 title: Get started - Build a bot
-author: heath-hamilton
+author: girliemac
 description: Quickly create a Microsoft Teams bot using the Microsoft Teams Toolkit.
-ms.author: lajanuar
-ms.date: 11/04/2020
+ms.author: timura
+ms.date: 04/14/2020
 ms.topic: tutorial
 ---
 # Build a bot for Microsoft Teams
 
-You'll build a basic *bot* app in this tutorial. A bot acts as an intermediary between Teams users and your web service. People can chat with a bot to quickly get information or initiate workflows and tasks performed by your service.
+This tutorial teaches you to build a basic bot app. A bot acts as an intermediary between Teams users and your web service with a conversational interface. People can chat with a bot to quickly get information or initiate workflows and tasks performed by your service.
 
-## Your assignment
+**This tutorial teaches you to:**
```

Comment:
```suggestion
## What you'll learn
```
In general, from a semantic value and search/findability standpoint, it's better to use headings than bold body text.